spaceship/lib/spaceship/portal/ui/select_team.rb

```diff
@@ -32,9 +32,15 @@ class UserInterface
       #   ]
 
       def select_team
-        teams = client.teams
-
-        raise "Your account is in no teams" if teams.count == 0
+        begin
+          teams = client.teams
+          raise "Your account is in no teams" if teams.count == 0
```

Comment:
My thinking:

* client.teams can raise an exception on its own (the original crash) and…
* teams.count == 0 is an exception of the same variety

Check and catch both here with the appropriate rescue help text.